pynest/nest/tests/test_poisson_generator_rate_change.py

```diff
@@ -0,0 +1,85 @@
+# -*- coding: utf-8 -*-
+#
+# test_poisson_generator_rate_change.py
+#
+# This file is part of NEST.
+#
+# Copyright (C) 2004 The NEST Initiative
+#
+# NEST is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 2 of the License, or
+# (at your option) any later version.
+#
+# NEST is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with NEST.  If not, see <http://www.gnu.org/licenses/>.
+
+
+import nest
+import unittest
+import scipy.stats
+import numpy as np
+
+
+class TestPgRateChange(unittest.TestCase):
+
+    def test_statistical_rate_change(self):
+        """Statistical test of poisson_generator_ps rate change"""
+
+        # Lower limit of acceptable p_value
+        p_value_lim = 0.3
+
+        def kstest_first_spiketimes(sd, start_t):
+            events = nest.GetStatus(sd)[0]['events']
+            senders = events['senders']
+            times = events['times']
+            min_times = [np.min(times[np.where(senders == s)])
+                         for s in np.unique(senders)]
+            d, p_val = scipy.stats.kstest(
+                min_times, 'expon', args=(start_t + 0.1, 10.))
```

Comment:
I am not really happy with the "magic numbers" 0.1 and 10. here. It would be more robust and transparent to pass the expected rate  as argument. For even more robustness, I'd suggest to set the resolution explicitly and have it in a variable; maybe use 0.25 or so to avoid the numerically unpleasant 0.1 ;).